src/Http/Http/src/ReadOnlyPipeStream.cs

```diff
@@ -11,6 +11,7 @@ namespace System.IO.Pipelines
     /// <summary>
     /// Represents a read-only Stream backed by a PipeReader
     /// </summary>
+    // REVIEW: Could this be replaced or simplified with https://apisof.net/catalog/System.IO.Pipelines.PipeReader.AsStream() ?
```

Comment:
It really should be able to. What issues did you run into here?

Reply:
I didn't run in to any issues, I just didn't know exactly what this type was for :)

Reply:
I can give it a try.

Reply:
Oh nvrm I see the issue. Kestrel has a class that derives this. The stream types in corefx aren't public.

Reply:
Leave these types for now